package.json

```diff
@@ -1,28 +1,29 @@
 {
+  "version": "2.3.0",
   "scripts": {
-    "start": "expo start --dev-client",
-    "android": "expo run:android",
-    "ios": "expo run:ios",
-    "web": "expo start --web",
+    "android": "react-native run-android",
+    "ios": "react-native run-ios",
     "lint": "eslint .",
-    "postinstall": "jetify"
+    "start": "react-native start",
+    "test": "jest"
   },
   "dependencies": {
     "@react-native-async-storage/async-storage": "^1.17.10",
     "@react-native-community/geolocation": "^2.1.0",
     "@react-native-community/push-notification-ios": "^1.10.1",
     "appcenter": "4.4.5",
     "appcenter-crashes": "4.4.5",
-    "bluedot-react-native": "Bluedot-Innovation/Bluedot-React-Native-Plugin#release/2.2.1",
-    "expo": "~46.0.7",
-    "expo-splash-screen": "~0.16.1",
-    "react": "18.0.0",
+    "bluedot-react-native": "Bluedot-Innovation/Bluedot-React-Native-Plugin#dn/react-0.69-above",
```

Comment:
will update to branch `release/2.3.0` once Plugin MR is merged. Will also update to the npmjs published one once we release plugin 2.3.0 to the public.